ais_service_discovery/call_service.py

```diff
@@ -23,9 +43,11 @@
 sqs_adaptor = SqsAdaptor(sqs)
 send = Send(sqs_adaptor)
 
-def run_service(service, body, opts={}):
-    type = service['Attributes']['type']
-    if type in ['cloud-function', 'function', 'lambda']:
+
+def run_service(service, body, opts):
```

Comment:
```suggestion
def run_service(service, body, opts={}):
```

Reply:
Hmmm my py linter was kicking off about this, but I'll add it back in